src/test/run-pass/allocator-default.rs

```diff
@@ -10,7 +10,7 @@
 
 #![feature(alloc_jemalloc)]
 
-#[cfg(any(target_os = "linux", target_os = "macos"))]
+#[cfg(any(all(target_os = "linux", not(any(target_arch = "aarch64", target_arch = "mips", target_arch = "mips64", target_arch = "powerpc", target_arch = "powerpc64", target_arch = "s390x"))), target_os = "macos"))]
```

Comment:
What is the point of a testsuite if you're not running it? If users of non-x86 platforms are supposed to disable the testsuite, they shouldn't be really trusting any of the generated binaries, to be honest.